ROMFS/px4fmu_common/init.d/rc.thermal_cal

```diff
@@ -0,0 +1,35 @@
+#!nsh
+#
+# Thermal Calibration startup script.
+#
+# NOTE: Script variables are declared/initialized/unset in the rcS script.
+#
+
+#
+# Check if we should start a thermal calibration
+#
+if param compare SYS_CAL_GYRO 1
+then
+	set TEMP_CALIB_ARGS "-g"
+	param set SYS_CAL_GYRO 0
+fi
+
+if param compare SYS_CAL_ACCEL 1
+then
+	set TEMP_CALIB_ARGS "-a"
```

Comment:
This needs to be `set TEMP_CALIB_ARGS "${TEMP_CALIB_ARGS} -a"`, otherwise you cannot do multiple calibrations in one go. Can you revert to the previous logic?
Also `TEMP_CALIB_ARGS` can be local in this script.

Reply:
Thanks @bkueng ,

I hadn't considered multiple calibration routines in one go and the proposed changes indeed broke that functionality.  To your point in making the script variable local, although leaving it in the rcS script does allow setting/unsetting `TEMP_CALIB_ARGS` at the config file level as well, for this PR it certainly makes sense to deal with a minimal change set.

Changes addressing your comment are pushed up and rebased against current upstream/master.  Thanks for the feedback!

-Mark

Reply:
> To your point in making the script variable local, although leaving it in the rcS script does allow setting/unsetting TEMP_CALIB_ARGS at the config file level as well, for this PR it certainly makes sense to deal with a minimal change set.

For this particular variable I don't think it needs to be set anywhere else. Keeping it local reduces complexity.